### PR TITLE
Make the installation repos view always consistent

### PR DESCRIPTION
**Pull request by dennwc**
Previously the view for repository installations was eventually consistent. The list of repository IDs is always consistent, however, we used an `INNER JOIN` with a repository nodes which is only available once data is loaded. Thus, the view was missing repos that were not yet processed and made the API unhappy.

To fix it, this change adds enough information to the repository installation table to avoid `JOIN`s with eventually-consistent data.

This is a breaking change and it requires a migration. It was already applied to all our environments to mitigate the effect of the issue.

Closes [ENG-613] on metadata side for future installations (current ones are already OK).

[ENG-613]: https://athenianco.atlassian.net/browse/ENG-613

### CODE REVIEW

File: github/schema.sql [hunk 1]
```diff
@@ -46,6 +46,7 @@ CREATE TABLE IF NOT EXISTS public.github_installation_owners (
 CREATE TABLE IF NOT EXISTS public.github_installation_repos (
     install_id bigint NOT NULL REFERENCES public.github_installations(id) ON DELETE CASCADE,
     repo_id bigint NOT NULL,
+    repo_node_id text NOT NULL UNIQUE,
```
Comment (vmarkovtsev) [inline github/schema.sql:49]:
```suggestion
    repo_id text NOT NULL UNIQUE,
```

I don't know why but the API is already thinking that `repo_id` is the text node ID.

Comment (dennwc) [reply github/schema.sql:49]: Note that you are commenting on the _table_, not a _view_ that you consume ;)
Here the `repo_id` is similar to a `database_id` (an integer), while `repo_node_id` will be the node ID (a string).
View will still have a `repo_id` which is a string.

**Comment by dennwc** [comment]
Assuming it's okay. Merging.

File: github/schema.sql (hunk 1 above)
Comment (vmarkovtsev) [reply github/schema.sql:49]: Aaaaah OK

**Pull request merged**